aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/featuretest/generators/testcasegenerators/ResetPasswordTestCaseGenerator.kt

```diff
@@ -96,32 +95,5 @@ object ResetPasswordTestCaseGenerator : SerializableProvider {
         validations = baseCase.validations.plus(apiReturnValidation)
     )
 
-    private val errorCase: FeatureTestCase
```

Comment:
Can we fix the error case instead of removing it?

Reply:
Done